client/state/media/reducer.js

```diff
@@ -439,6 +441,46 @@ export const transientItems = withoutPersistence(
 	}
 );
 
+/**
+ * Stores media actions meant to be passed on to Gutenframe
+ *
+ * Note: we used to receive plain action objects for deleted and updated
+ * media items from the Flux store. However, after the migration to Redux
+ * this isn't the case anymore. This is why we created this "channel"
+ * to hold media actions objects including the media item itself and the
+ * corresponding action ('deleted' or 'updated'). It's the responsibility
+ * of the CalypsoifyIFrame component to consume those and remove them after
+ * it passed them to the IFrame. Make sure the editor iframe is loaded before
+ * you add action items here.
+ *
+ * @param  {object} state  Current state
+ * @param  {object} action Action payload
+ * @returns {object}        Updated state
+ */
+export const _gutenframeMediaActions = withoutPersistence( ( state = {}, action ) => {
```

Comment:
Do we really need the leading underscore? Let's remove it if there's no good need for it.